### PR TITLE
Migrate connections guide

### PR DESCRIPTION
**Pull request by kwcanuck**
Migration of the existing guide connections (Managing your Connections in Apache Airflow)

### CODE REVIEW

File: guides/connections.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+---
+title: "Manage Connections in Apache Airflow"
+sidebar_label: "Manage Apache Airflow connections"
```
Comment (jwitz) [inline guides/connections.md:3]:
```suggestion
sidebar_label: "Connections"
```

File: guides/connections.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
+---
+title: "Manage Connections in Apache Airflow"
```
Comment (jwitz) [inline guides/connections.md:2]:
```suggestion
title: "Manage connections in Apache Airflow"
```

File: guides/connections.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+# an example of a connection to snowflake defined as a URI
+ENV AIRFLOW_CONN_SNOWFLAKE_CONN='snowflake://LOGIN:PASSWORD@/?account=xy12345&region=eu-central-1'
+
+```
+
+In Airflow 2.3+, connections can be provided to an environment variable as a JSON dictionary:
+
+```json
+# example of a connection defined as a JSON file in your `.env` file
+AIRFLOW_CONN_MYCONNID='{
```
Comment (jwitz) [inline guides/connections.md:96]: Not sure why these are showing up bright red, but I find that deleting/ re pasting the problematic text usually solves the issue.

**Review by jwitz — APPROVED** [review]

File: guides/connections.md (hunk 3 above)
Comment (kwcanuck) [reply guides/connections.md:96]: @jwitz the issue appears to be resolved.

File: guides/connections.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+![Connections seen from the DAG view](/img/guides/DAGview_connections_cropped.png)
+
+Airflow doesn't provide any preconfigured connections. To create a new connection, click the blue **+** button.
+
+![Empty Connection](/img/guides/EmptyConnection.png)
+
+As you update the **Connection Type** field, notice how the other available fields change. Each connection type requires different kinds of information.
+
+> **Note**: Specific connection types are only available in the dropdown menu if the relevant provider is installed in your Airflow environment.
```
Comment (jwitz) [inline guides/connections.md:62]: Turn this into an `:::info` note

File: guides/connections.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+Any parameters that don't have specific fields in the connection form can be defined in the **Extra** field as a JSON dictionary. For example, you can add the `sslmode` or a client `sslkey` in the **Extra** field of your PostgreSQL connection.
+
+Starting in Airflow 2.2, you can test some connection types from the Airflow UI with the **Test** button. After running a connection test, a message appears on the top of the screen showing either a success confirmation or an error message.
+
+## Define connections with environment variables
+
+Connections can also be defined using environment variables. If you use the Astro CLI, you can use the `.env` file for local development or specify environment variables in your project's Dockerfile.
+
+> **Note**: If you are synchronizing your project to a remote repository, don't save sensitive information in your Dockerfile. In this case, using either a secrets backend, Airflow connections defined in the UI, or `.env` locally are preferred to avoid exposing secrets in plain text.
```
Comment (jwitz) [inline guides/connections.md:76]: Turn this into a caution note? Or maybe not a note at all

File: guides/connections.md (hunk 3 above)
Comment (jwitz) [reply guides/connections.md:96]: We can check it in the base PR to make sure it renders correctly, I wouldn't worry about it for now. Might just be GH freaking out

**Pull request merged**